integrations/auth.go

```diff
@@ -0,0 +1,11 @@
+package integrations
+
+// Align these with:
+// https://github.com/autokitteh/web-platform/blob/main/src/enums/connections/connectionTypes.enum.ts
+const (
+	APIToken   = "apiToken"
+	JSONKey    = "jsonKey" // Google integrations only
+	OAuth      = "oauth"
+	PAT        = "pat"
+	SocketMode = "socketMode" // Slack integration only
```

Comment:
I noticed that not all types found [here](https://github.com/autokitteh/web-platform/blob/main/src/enums/connections/connectionTypes.enum.ts) are represented. Is this intentional?

Reply:
Yes, the list in this PR is (or rather will be) the authoritative one. The TS file is tentative.